spec/lib/integrations/slack_spec.rb

```diff
@@ -175,65 +175,4 @@
       end
     end
   end
-
-  describe '#message_color' do
```

Comment:
Can you add specs test the color is correct for the new way you are determining it?